[web-sys] yew 0.13 fails to compile

**Problem**
<!-- A clear and concise description of what the bug is. -->

I want to update my project from std_web to web_sys.
When trying to build an `empty` (or not) project with the new feature `web_sys`, yew 0.13 does not compile.

**Steps To Reproduce**
Steps to reproduce the behavior:
1. `cargo new --lib project`
2. Cargo.toml
```yml
[package]
name = "project"
version = "0.1.0"
edition = "2018"

[dependencies]
yew = { version = "0.13", features = ["web_sys"] }

[lib]
crate-type = ["cdylib", "rlib"]
```
3. src/lib.rs
```rust
fn main() {
    println!("Hello, world!");
}
```
4. Run `cargo wasm build` or `wasm-pack build` (it will be the same error with both)
5. See error
```bash
   Compiling yew v0.13.0
error[E0308]: mismatched types
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/yew-0.13.0/src/virtual_dom/vtext.rs:113:9
    |
60  |       ) -> Option<Node> {
    |            ------------ expected `std::option::Option<web_sys::features::gen_Node::Node>` because of return type
...
113 | /         self.reference.as_ref().map(|t| {
114 | |             let node = cfg_match! {
115 | |                 feature = "std_web" => t.as_node(),
116 | |                 feature = "web_sys" => t.deref(),
117 | |             };
118 | |             node.to_owned()
119 | |         })
    | |__________^ expected struct `web_sys::features::gen_Node::Node`, found struct `web_sys::features::gen_CharacterData::CharacterData`
    |
    = note: expected enum `std::option::Option<web_sys::features::gen_Node::Node>`
               found enum `std::option::Option<web_sys::features::gen_CharacterData::CharacterData>`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: could not compile `yew`.

To learn more, run the command again with --verbose.
Failed to run cargo. Exit code was: 101
```

**Expected behavior**
I was expecting yew to build, but maybe am I doing something wrong ?

**Environment:**
 - Yew v0.13
 - Rust v1.41.1
 - Target wasm32-unknown-unknown
 - web-sys v0.3.36
 - OS: ubuntu

--- a/src/virtual_dom/vtext.rs
+++ b/src/virtual_dom/vtext.rs
@@ -113,7 +113,7 @@ impl VDiff for VText {
         self.reference.as_ref().map(|t| {
             let node = cfg_match! {
                 feature = "std_web" => t.as_node(),
-                feature = "web_sys" => t.deref(),
+                feature = "web_sys" => t.deref().deref(),
             };
             node.to_owned()
         })

